config/manager/kustomization.yaml

```diff
@@ -1,10 +1,23 @@
 resources:
 - manager.yaml
+- workflows/kfp/pipeline-steps.yaml
+- workflows/kfp/create-pipeline.yaml
+- workflows/kfp/update-pipeline.yaml
+- workflows/kfp/delete-pipeline.yaml
+- workflows/kfp/experiment-steps.yaml
+- workflows/kfp/create-experiment.yaml
+- workflows/kfp/update-experiment.yaml
+- workflows/kfp/delete-experiment.yaml
+- workflows/kfp/runconfiguration-steps.yaml
+- workflows/kfp/create-runconfiguration.yaml
+- workflows/kfp/update-runconfiguration.yaml
+- workflows/kfp/delete-runconfiguration.yaml
```

Comment:
We could have all resources related to one CRD in one file to reduce the number of files